src/ert/shared/models/single_test_run.py

```diff
@@ -11,22 +12,18 @@ def __init__(
         self, simulation_arguments: Dict[str, Any], ert: EnKFMain, id_: str, *_: Any
     ):
         local_queue_config = ert.get_queue_config().create_local_copy()
-        super().__init__(simulation_arguments, ert, local_queue_config, id_)
+        super().__init__(id_, simulation_arguments, ert, local_queue_config)
```

Comment:
I don't really get it, how has this not failed before? The positional arguments were incorrect??

Reply:
As far as I understand, ´super.__inti__´ refers to 
```
class EnsembleExperiment(BaseRunModel):
    def __init__(
        self,
        id_: str,
        simulation_arguments: Dict[str, Any],
        ert: EnKFMain,
        queue_config: QueueConfig,
    ):
```

and the order was incorrect there.
Is it a problem to have it in the same order as the super class?